### PR TITLE
Updated wording for 404 response handler. Issue #917.

### PR DESCRIPTION
**Pull request by nickgrealy**
I was only able to provide the updated wording for English based docs.

### CODE REVIEW

**Review by nickgrealy — COMMENTED** [review]
Approved.

File: en/starter/faq.md [hunk 1]
```diff
@@ -58,7 +58,8 @@ because a 404 response simply indicates the absence of additional work to do;
 in other words, Express has executed all middleware functions and routes,
 and found that none of them responded. All you need to
 do is add a middleware function at the very bottom of the stack (below all other functions)
-to handle a 404 response:
+to handle a 404 response. Adding routes dynamically at runtime, should be performed on
```
Comment (crandmck) [inline en/starter/faq.md]: I suggest wording such as this:
> Add routes dynamically at runtime on an instance of `express.Router()` so the routes are not superseded by a middleware function:

**Review by crandmck — DISMISSED** [review]

**Comment by dougwilson** [comment]
@nickgrealy any desire to update the PR?

File: en/starter/faq.md (hunk 1 above)
Comment (nickgrealy) [reply en/starter/faq.md]: Sounds good. I'm happy with that.

**Comment by nickgrealy** [comment]
@dougwilson / @crandmck - suggestions added. Let me know if there's anything else required (e.g. squash/rebase?)

**Review by dougwilson — APPROVED** [review]

**Pull request merged**